cg/server/dto/orders/orders_response.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Order(BaseModel):
-    customer_id: int
+    customer_id: str
```

Comment:
You probably want to add the pipeline here as well, right? 😄

Reply:
Yes! That would be ideal 😅